### ISSUE TITLE
Intro_to_GANs_Solution: d_loss_fake

### ISSUE THREAD

**Issue by leros** [udacity/deep-learning#126]
In `d_loss_fake = tf.reduce_mean(
                  tf.nn.sigmoid_cross_entropy_with_logits(logits=d_logits_fake, 
                                                          labels=tf.zeros_like(d_logits_real)))`

Since we use fake logits for labels of all zeros for discriminator, should we pass `d_logits_fake` in for the `labels`, e.g. ... ` labels=tf.zeros_like(d_logits_fake)`...?

**Comment by MrDNA2018** [comment]
Yes

**Comment by MrDNA2018** [comment]
![img_0165](https://user-images.githubusercontent.com/36729595/44842084-86f3a400-ac77-11e8-9a09-95548a220648.PNG)
When rescaling why "x=2*x -1"

**Comment by abhiojha8** [comment]
Though we are a little late to respond, we suggest you reach out to Udacity support for user-specific issues. 
Thanks
